README.md

```diff
@@ -1,4 +1,4 @@
-<img align="left" width="440" height="180" alt="Fastest full PostgreSQL nodejs client" src="https://raw.githubusercontent.com/porsager/postgres/master/postgresjs.svg?sanitize=true" />
+<img style="float: left;" width="440" height="180" alt="Fastest full PostgreSQL nodejs client" src="https://raw.githubusercontent.com/porsager/postgres/master/postgresjs.svg?sanitize=true">
```

Comment:
It needs to be `align="left"` to work with github markdown - see https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d